stl/inc/xutility

```diff
@@ -2110,12 +2110,13 @@ namespace ranges {
 
             template <class _Ty>
             _NODISCARD static _CONSTEVAL _Choice_t<_St> _Choose() noexcept {
+                _STL_INTERNAL_CHECK(is_lvalue_reference_v<_Ty>);

```

Comment:
`_STL_INTERNAL_CHECK` is runtime. I believe you should introduce an `_STL_STATIC_ASSERT`. (Occurs below.)

Reply:
I used `_STL_INTERNAL_STATIC_ASSERT`.